README.md

```diff
@@ -586,6 +586,130 @@ The accepted formats are:
 
 Check out `spec/*.spec.js` files for more examples!
 
+## Using multiple exported translation files on a page.
+This method is useful for very large apps where a single contained translations.js file is not desirable. Examples would be a global translations file and a more specific route translation file.
+
+### Rails without asset pipeline
+1. Setup your `config/i18n-js.yml` to have multiple files and try to minimize any overlap.
+  ```yaml
+  sort_translation_keys: true
+  fallbacks: false
+
+  translations:
+    + file: "app/assets/javascript/nls/welcome.js"
+      only:
+        + '*.welcome.*'
+      pretty_print: true
```

Comment:
@AntiFish03 
Maybe that's what you are using.
But `pretty_print` doesn't have to be true to have `multiple exported translation files`
Could you remove it to avoid confusion?